custom/fitness-demo/pages/[room].js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
```

Comment:
I am using a dynamic route here so that the user won't be losing the room which he is in when he refreshes the screen and would also be easy to invite users to the call.

Like let's say I joined a room called `demo` from the index page `/` so now I will be directed to `/demo` and here I will be in the call, so now if I refresh the page - I will still be in the same call

If we are not using dynamic routes then after the refresh I will be redirected to the landing page as it will lose the context.